### PR TITLE
osqp 0.5.0 (new formula)

### PR DESCRIPTION
**Pull request by jamiesnape**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/osqp.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
+class Osqp < Formula
+  desc "Operator splitting QP solver"
+  homepage "https://osqp.org/"
+  url "https://github.com/oxfordcontrol/osqp/archive/v0.5.0.tar.gz"
+  sha256 "e0932d1f7bc56dbe526bee4a81331c1694d94c570f8ac6a6cb413f38904e0f64"
+
+  depends_on "cmake" => :build
```
Comment (zbeekman) [inline Formula/osqp.rb]:
```suggestion
  depends_on "cmake" => [:build. :test]
```

You need a test dependency on CMake too.

**Review by zbeekman — CHANGES_REQUESTED** [review]

File: Formula/osqp.rb (hunk 1 above)
Comment (zbeekman) [reply Formula/osqp.rb]: I would push this myself if I had permission...

Comment (jamiesnape) [reply Formula/osqp.rb]: Done

**Review by zbeekman — APPROVED** [review]

**Comment by zbeekman** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @jamiesnape for the pull request!

**Pull request closed without merging**